framework/include/chaincontrols/LimitChainControl.h

```diff
@@ -0,0 +1,35 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "ChainControl.h"
+
+/**
+ * Limits a control value by a range.
+ */
+class LimitChainControl : public ChainControl
+{
+public:
+  static InputParameters validParams();
+
+  LimitChainControl(const InputParameters & parameters);
+
+  virtual void execute() override;
+
+protected:
+  /// Minimum value to apply to control data
+  const Real _min_value;
```

Comment:
would be nice to have these be references to ChainControls?

Reply:
It would, but I'd prefer that the user not be required to add chain controls for the bounds if they just want constant values. To get the best of both approaches, I think we'd need to do some work to allow constants to be able to be supplied for control data parameters, and I'd prefer to do that work when it becomes needed.